src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/SoulboundListener.java

```diff
@@ -42,8 +41,8 @@ public void onDamage(PlayerDeathEvent e) {
         for (int slot = 0; slot < p.getInventory().getSize(); slot++) {
             ItemStack item = p.getInventory().getItem(slot);
 
-            // Store soulbound items for later retrieval
-            if (SlimefunUtils.isSoulbound(item)) {
+            // Store soulbound items for later rtrieval
```

Comment:
How you butchered this is beyond me.
```suggestion
            // Store soulbound items for later retrieval
```